stable/metabase/values.yaml

```diff
@@ -72,6 +72,16 @@ ingress:
     # - secretName: metabase-tls
     #   hosts:
     #     - metabase.domain.com
+
+# A custom log4j.properties file can be provided using a multiline YAML string.
+# See https://github.com/metabase/metabase/blob/master/resources/log4j.properties
+#
+# Also don't forget to pass "-Dlog4j.configuration=file:/tmp/conf/log4j.properties"
+# value to javaToolOptions.
+# See https://github.com/metabase/metabase/blob/master/docs/operations-guide/start.md#configuring-logging-level
+#
+# log4jProperties:
+
```

Comment:
Wouldn't be better to "decorate" the javaToolOptions` value with the `-Dlog4j.configuration=file:...` flag automatically when the user specify a custom log4j properties file, instead of having to set two separate configurations to make a single change (customizing the log4j configuraiton, in this case)?

Reply:
Done in 47bb8e5